pkg/object/object.go

```diff
@@ -236,9 +295,85 @@ func (v *Volume) validate() error {
 	return nil
 }
 
+func (sd *SecretData) validate() error {
```

Comment:
Where do you validate if the field `Base64` is a valid base64 encoded value? 

Similarly for field `File`, it should be a valid path, either absolute or relative depending on what is supported!